README.md

```diff
@@ -69,15 +66,44 @@ qemu-system-aarch64 \
     -snapshot output/qcow2/disk.qcow2
 ```
 
-## Volumes
-- `/output` - used for output files
-- `/store` - used for the osbuild store
-- `/rpmmd` - used for the dnf-json rpm metadata cache
+## 📝 Arguments
+
+`bootc-image-builder` always requires one positional argument: The container image reference.
+
+The following tables show all optional arguments:
+
+| Argument     | Description                                                      | Default Value |
+|--------------|------------------------------------------------------------------|:-------------:|
+| **--config** | Path to a [build config](#-build-config)                         |       ❌       |
+| --tls-verify | Require HTTPS and verify certificates when contacting registries |    `true`     |
```

Comment:
Why isn't this bold? Or why are the other options bold? :)

Reply:
This might be a premature optimization, but I expect that the list of tiny options will grow, so I thought it would be nice to make the most important ones bold. I added a tiny sentence to explain why they are bold. :) Hope it helps.